deployment/build_and_release/modules/presubmit_cloudbuild_triggers/main.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  backend "gcs" {}
+}
+
+resource "google_service_account" "builder" {
+  account_id   = "cloudbuild-${var.env}"
+  display_name = "Armored Witness ${var.env} Builder Service Account"
+}
+
+resource "google_cloudbuild_trigger" "applet_release" {
+  location = "global"
+  # service_account = google_service_account.builder.id
```

Comment:
Will this be enabled in this PR? If not, please add a TODO. Likewise in the other files.

Reply:
Done.